README.md

```diff
@@ -2,12 +2,13 @@
 
 The following are alive Mastodon relay list.
 
-- Mastodon.host - https://relay.mastodon.host/inbox
-- Yiff.social - https://yiff.social/inbox
-- Moe.cat - https://mastodon-relay.moew.science/inbox
+- Mastodon.host - https://relay.mastodon.host
+- Yiff.social - https://yiff.social
+- Moew Relay (Chinese Only) - https://mastodon-relay.moew.science
```

Comment:
```suggestion
- [Moew Relay (Chinese Only)](https://mastodon-relay.moew.science) - https://mastodon-relay.moew.science/inbox
```

Why do you need to delete "/inbox"? Add other "/inbox" back, so you can copy and paste directly. You can add a link to the brand without '/inbox'.